code/game/objects/explosion.dm

```diff
@@ -246,4 +246,18 @@
 		T.color = null
 		T.maptext = ""
 
-
+proc/dyn_explosion(turf/epicenter, power, flash_range, adminlog = 1, ignorecap = 1, flame_range = 0 ,silent = 0, smoke = 1)
+	if(!power)
+		return
+	var/range = 0
+	range = round((2 * power)**DYN_EX_SCALE)
+	explosion(epicenter, round(range * 0.25), round(range * 0.5), round(range), flash_range*range, adminlog, ignorecap, flame_range*range, silent, smoke)
+
+// Using default dyn_ex scale:
```

Comment:
its just the 

heavy = 1/4, mid = 1/2, light = 1  that's used everywhere else though.